### ISSUE TITLE
Transparency not coming through in color from Figma

### ISSUE THREAD

**Issue by chrisdrackett** [diez/diez#50]
I have a color with transparency in Figma (white at 90% transparent) that on extract comes through into Diez at `Color.rgba(255, 255, 255, 1)`. I would expect `Color.rgba(255, 255, 255, 0.9)`

**Comment by zackbrown** [comment]
Thanks for the report @chrisdrackett . I’ve slotted this for investigation with our next sprint. This may happen after the holidays.

To ease reproduction: could you please share a figma file we can refer to? e.g. a simplified copy or outright copy of the one where you’re encountering this issue.

**Comment by chrisdrackett** [comment]
here is a test file: https://www.figma.com/file/Q0xJJmMb4KlvevhTjXsIZM/Transparency?node-id=0%3A1

and here is what it outputs on extract:

```
/**
 * This code was generated by Diez version 10.1.0.
 * Changes to this file may cause incorrect behavior and will be lost if the code is regenerated.
 */
import { Color } from "@diez/prefabs";

const transparencyColors = {
    halfTransparent: Color.rgba(255, 255, 255, 1)
};

export const transparencyTokens = {
    colors: transparencyColors
};
```

**Comment by zackbrown** [comment]
Awesome, thanks! We’ll follow up here.

**Comment by roperzh** [comment]
Fixed for solids (this use case) in #55. Blend modes for gradients coming soon.
